fs/nfs/nfs_vfsops.c

```diff
@@ -1213,6 +1215,84 @@ static ssize_t nfs_write(FAR struct file *filep, FAR const char *buffer,
   return byteswritten > 0 ? byteswritten : ret;
 }
 
+/****************************************************************************
+ * Name: nfs_seek
+ *
+ * Returned Value:
+ *   Returns the resulting offset location as measured in bytes from the
+ *   beginning of the file.
+ *
+ ****************************************************************************/
+
+static off_t nfs_seek(FAR struct file *filep, off_t offset, int whence)
+{
+  FAR struct nfsmount *nmp;
+  FAR struct nfsnode  *np;
+  off_t                ret;
+
+  /* Sanity checks */
+
+  DEBUGASSERT(filep->f_priv != NULL && filep->f_inode != NULL);
+
+  /* Recover our private data from the struct file instance */
+
+  nmp = (FAR struct nfsmount *)filep->f_inode->i_private;
+  np  = (FAR struct nfsnode *)filep->f_priv;
+
+  DEBUGASSERT(nmp != NULL);
+
+  ret = nxmutex_lock(&nmp->nm_lock);
+  if (ret < 0)
+    {
+      return ret;
+    }
+
+  /* Map the offset according to the whence option */
+
+  switch (whence)
+    {
+      case SEEK_SET:
+          break;
```

Comment:
remove 2 extra spaces here and other places

Reply:
@pkarashchenko  thanks, fix with PR https://github.com/apache/nuttx/pull/9982